Sources/HaCWebsiteLib/Config.swift

```diff
@@ -24,4 +24,9 @@ public struct Config {
   public static func checkEnvVars() {
     checkEnvVarsExist(for: "DATA_DIR")
   }
+
+  public static var isProduction: Bool {
```

Comment:
We can use this now to silence compiler output on production now can't we?

Reply:
Should I create an issue for this?

Reply:
@moosichu yeah go ahead!